### DISCUSSION TITLE
Incognito workaround

### DISCUSSION THREAD

**Discussion by thecoolmiker** [3kh0/ext-remover#737] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Just found out that if you sign out with the dns/wifi using the incognito dns then on the login screen you click back, then you can click sign in under the network part, . then double click webview test, itll open a tab without you even being logged in, no filter no exts or anything
![image](https://github.com/3kh0/ext-remover/assets/100372901/a06ca756-1c30-40f3-9c9a-4018bcc16154)

**Comment by funkymokey** [comment]
This is the most recent chrome os version?

**Reply by thecoolmiker** [reply]
Yes

**Comment by 3kh0** [comment]
Nice

**Comment by MechaXYZ** [comment]
it works but it has no cursor lock so I can't play geforce now 💀

**Reply by thecoolmiker** [reply]
That is a con i dislike
